app/settings/agents_variables.yaml

```diff
@@ -1,4 +1,5 @@
 ICTRTS:
+  validation_size: 0.1
```

Comment:
is this right?

Reply:
No, we forgot to remove it, it's just an idea